test/github_pullrequest_test.go

```diff
@@ -12,23 +12,20 @@ import (
 )
 
 func TestGithubPullRequest(t *testing.T) {
-	t.Parallel()
```

Comment:
t.Parallel actually makes it parallel isnt it ?

Reply:
yeah but one we use same repo right
so once test created repo
other will also try to create and fail in a different ns

Reply:
👍🏻